.github/workflows/codeql-analysis.yml

```
@@ -59,27 +59,9 @@ jobs:
-        project: 'EventRelay'
-        path: '.'
-        format: 'SARIF'
-        args: >-
-          --enable-retired
-          --exclude './frontend/node_modules,./src/mcp-bridge.py'
-    
-    - name: Upload dependency check results
-      uses: github/codeql-action/upload-sarif@v4
-      with:
-        sarif-file: 'reports/dependency-check-report.sarif'
+  # Dependency / supply-chain scanning is handled by dedicated workflows:
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)